pkg/app/instances/gce.go

```diff
@@ -61,6 +61,26 @@ func NewGCEInstanceManager(cfg Config, service *compute.Service, nameGenerator N
 	}
 }
 
+func (m *GCEInstanceManager) ListZones() (*apiv1.ListZonesResponse, error) {
+	res, err := m.Service.Zones.List(m.Config.GCP.ProjectID).Context(context.TODO()).Do()
+
+	if err != nil {
+		return nil, toAppError(err)
+	}
+
+	var items []*apiv1.Zone
+	for _, i := range res.Items {
+		hi, err := BuildZone(i)
+		if err != nil {
+			return nil, err
+		}
+		items = append(items, hi)
```

Comment:
If there is no possibility of errors while converting compute.Zone to apiv1.Zone, how about changing these lines to 

```
items = append(items, &apiv1.Zone{ Name: item.Name })
```

and remove BuildZone() function.